2021/Application.md

```diff
@@ -0,0 +1,68 @@
+# Application process
+Technical writers interested in participating to GSoD with ArviZ should follow these steps:
+
+1. Join the dedicated ArviZ Gitter channel for season of docs: https://gitter.im/arviz-devs/season_of_docs
+2. Send an statement of interest to `arviz.devs@gmail.com` before May 1st. More info about your statement of
+   interest at []().
+3. Fulfill the application requirements before May 10th.
+4. Be available for feedback on your proposal.
+
+## Application requirements
+Only applications from people who perform the couple tasks below will be considered.
+
+The only requirement will be ...
+
+---
+---
+
+# Selection criteria
+As ArviZ project we strive to both share statistical ideas to aid the scientific workflow and to provide the code that enables that workflow.
+We’re looking for someone who shares our interest in communicating those ideas with clear and concise communication.
+
+## Required skills
+We believe that these three skills are necessary to be successful in the GSoD project:
```

Comment:
Theres four skills here, but perhaps we move the bottom one, I dont think its required that they have knowledge of a documentation framework, but it would be preferred

Reply:
I think knowing about a documentation framework, any documentation framework, not necessarily diataxis or gooddocsproject ones is an imprescindible skill to successfully execute the project. I guess they could learn about them during the project, but , 2 big buts, unlike git/sphinx/programming, I am not sure we have enough expertise to teach documentation frameworks, and also, IMO the main added value technical writers can provide is knowledge like documentation frameworks, it's getting specific feedback and extra insight that we would not get if we hired ourselves for the job

Reply:
Ok Im sold. Lets change the required skill number to four :)

Reply:
ooh, we should be extra clear on that in the requirements then to make sure technical writers are not confused on that too

Reply:
While writing this im like im so bad at writing im glad were hiring a technical writer 😓

Reply:
same here

Reply:
We definitively need to explain what we mean by "documentation framework"